FeaturePoints/src/cz/fidentis/featurepoints/pdm/CSVparser.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.fidentis.featurepoints.pdm;
```

Comment:
preco je to tu? je to to iste ako parser, ktory uz vo Fidentise je? Ak ano, tak uprav/pouzij ten, ak nie, tak daj tento parser tam, kde su ostatne parsre z Fidentisu